node/pkg/adminrpc/adminserver_test.go

```diff
@@ -120,8 +120,20 @@ func generateMockVAA(gsIndex uint32, gsKeys []*ecdsa.PrivateKey) []byte {
 		EmitterAddress:   vaa.Address{},
 		Payload:          []byte("test"),
 	}
-	for i, key := range gsKeys {
-		v.AddSignature(key, uint8(i))
+	for i, signer := range signers {
+		sig, err := signer.Sign(v.SigningDigest().Bytes())
+		if err != nil {
+			panic(err)
+		}
```

Comment:
panicing like this from within a unit test is poor form, please don't do this! Just use testify with [require.NoError(err)](https://pkg.go.dev/github.com/stretchr/testify/assert#NoError). Also, pass in a `*testing.T` to this function and in the very first line, do something like: [t.Helper()](https://pkg.go.dev/testing#T.Helper)